.github/workflows/docs-spelling-checks.yml

```diff
@@ -13,20 +13,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - name: Install aspell
-        run: sudo apt-get install aspell aspell-en
+      - name: Install vale
+        run: sudo snap install vale
       - id: spell-check
         name: Spell Check
-        run: make spelling
+        run: vale --glob='*.{md,txt,rst}' .
         working-directory: docs/canonicalk8s
-        continue-on-error: true
+      #  continue-on-error: true
       # - if: ${{ github.event_name == 'pull_request' && steps.spell-check.outcome == 'failure' }}
-      #   uses: actions/github-script@v6
-      #   with:
-      #     script: |
-      #       github.rest.issues.createComment({
-      #         issue_number: context.issue.number,
-      #         owner: context.repo.owner,
-      #         repo: context.repo.repo,
-      #         body: 'Hi, looks like pyspelling job found some issues, you can check it [here](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})'
-      #       })
+        # uses: actions/github-script@v6
```

Comment:
Is this commented because we are waiting on a green light from documentation folks to tun on the bot?